hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -154,11 +157,16 @@ Setup links for S3 tests
     Execute            ozone sh bucket create --layout ${BUCKET_LAYOUT} o3://${OM_SERVICE_ID}/legacy/source-bucket
     Create link        link
 
-Create generated bucket
+Create generated bucket with OBS
     [Arguments]          ${layout}=OBJECT_STORE
     ${BUCKET} =          Create bucket with layout    ${layout}
     Set Global Variable   ${BUCKET}
 
+Create generated bucket with FSO
+    [Arguments]          ${layout}=FILE_SYSTEM_OPTIMIZED
+    ${BUCKET1} =         Create bucket with layout    ${layout}
+    Set Global Variable   ${BUCKET1}
+
```

Comment:
As we have 2 different key words for creating FSO and OBS buckets, there is no need to pass an argument to these keywords. `[Arguments]          ${layout}=FILE_SYSTEM_OPTIMIZED` can be ommitted. 
Instead of passing `${layout}` to ` Create bucket with layout `, we can directly mention the layouts ${BUCKET_LAYOUT_OBS} and ${BUCKET_LAYOUT_FSO} respectively.